compiler-plugin/src/main/kotlin/arrow/meta/plugins/proofs/phases/ir/ProofsIrCodegen.kt

```diff
@@ -59,16 +72,15 @@ class ProofsIrCodegen(
       }
     }
   }
-
+  //TODO handle ambiguity and orphan selection
```

Comment:
if ambiguity and orphan selection has been done in the resolution phase then in IR is already guaranteed to not be ambiguous and we can remove this TODO
```suggestion
```